scifin/classifier/classifier.py

```diff
@@ -147,48 +153,57 @@ def kmeans_base_clustering(corr, names_features=None, max_num_clusters=10, n_ini
     -----
       Function adapted from "Machine Learning for Asset Managers",
       Marcos López de Prado (2020).
+
+      To learn more about sklearn.cluster.KMeans():
+      https://scikit-learn.org/stable/modules/generated/sklearn.cluster.KMeans.html#sklearn.cluster.KMeans
     """
     
     # Checks
     if not isinstance(max_num_clusters, int):
         raise AssertionError("max_num_clusters must be integer.")
-    if not isinstance(n_init, int):
-        raise AssertionError("n_init must be integer.")
     
     # Initializations
     corr = pd.DataFrame(data=corr, index=names_features, columns=names_features)
     silh_score = pd.Series()
     
-    # Define the observations matrix X
+    # Define the observations matrix
     X = ((1 - corr.fillna(0))/2.)**.5
```

Comment:
as a good coding practice, try give each variable a sensible name so it can be easily understood and refer back what it means later

Reply:
True. This one is not explicit, but it's quite common to feed a classifier.fit() function with X. Since I'm more concerned actually with my reuse of the symbol X for the observations matrix and then for the Euclidean distance matrix, I am going to clean this part.